### DISCUSSION TITLE
Translation missing error

### DISCUSSION THREAD

**Discussion by emrhnksck** [railsadminteam/rails_admin#3593] [Q&A] [answered]
**Describe the bug**
Hi i installed my project rails admin 3. I gave labels for grouping my models in rails_admin.rb. Labels comes from in en.yml file (en.admin.misc.example) but in admin panel rails admin gives error translation missing: en.admin.misc.example instead of my labels. I added the required fields to the en.yml file but it still gives an error how can i fix this?

**Expected behavior**
It should be group by my models acording the labels existing in en.yml file shouldnt gives translation missing error

**Additional context**

- `rails` version: 7.0.4
- `rails_admin` version: 3.1.1
- `rails_admin` npm package version: -

**Comment by mshibuya** [comment]
Could you be more specific, like providing the real source code?

**Comment by emrhnksck** [comment]
This is a private repo so i cannot share the real source code but  the code below is almost exactly the same. @mshibuya 

**en.yml**
```
en:
  admin:
    misc:
      example1: example11
      example2: example22
      example3: example33
```

**rails_admin.rb**
```
  example1_label = I18n.t('admin.misc.example1')
  example2_label = I18n.t('admin.misc.example2')
  example3_label = I18n.t('admin.misc.example3')
```

```
  config.model 'Example' do
    navigation_label example1_label
                      .
                      .
                      .
```

**Comment by mshibuya** [comment, marked as answer]
Due to the Rails initialization order, locales are not loaded upon the initializer execution.
Use this style to get it lazily evaluated.

```ruby
navigation_label { I18n.t('admin.misc.example1') }
```

**Reply by emrhnksck** [reply]
This solved my problem thank you so much!
